app/main/views/frameworks.py

```diff
@@ -250,8 +250,9 @@ def get_framework_suppliers(framework_slug):
             cfa.status.in_(status.split(","))
         )
 
+    with_users = convert_to_boolean(request.args.get("with_users", default="false"))
```

Comment:
I don't think this flag should be here if it's not used by any of the frontend apps. This listing is very expensive, so if it's not actually used then there's no reason to have it here.